### DISCUSSION TITLE
add languages

### DISCUSSION THREAD

**Discussion by pedroperez90** [openbullet/openbullet#801] [Ideas]
How languages ​​could be added, so that the interface is more friendly.

**Comment by openbullet** [comment]
This is OpenBullet 1, it is not supported anymore, and adding languages has never been supported unless you manually translate everything from English to your language (I've seen someone do it for Spanish). OpenBullet 2 instead supports localization, if you can't find your language and you want to contribute you can do that. Check out this guide https://discourse.openbullet.dev/t/how-to-add-or-modify-translations/33
